packages/retail-ui-extensions/src/extension-api/smartgrid-api/smartgrid-api.ts

```diff
@@ -0,0 +1,11 @@
+export interface SmartGridApiContent {
+  /** Opens a full screen with the Retail::SmartGrid::Modal extension point */
+  navigateToFullScreenModal(): void;
```

Comment:
What do you think if we would rename this to `presentModal`?

Reply:
Yeah that sounds better and is easier to understand